examples/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -508,4 +514,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = 97C146E61CF9000F007C117D /* Project object */;
-} 
\ No newline at end of file
+}
```

Comment:
?

Reply:
Xcode just auto-adds final newlines

Reply:
I didn't mean the newline, I meant all the changes to this file. They seem unrelated to the patch.

Reply:
Oh I see. Ya, it's somewhat unrelated. Leftovers from https://github.com/flutter/flutter/pull/13195 which added new plugins. We'll just get these diffs if anyone flutter runs on iOS until we check them in.